doc/release-notes.md

```diff
@@ -1,6 +1,126 @@
 # release notes
 
 ## current release
+- release date: 2022-06-21
+- status: available
+- overview of changes:
+  - This particular release added the following datasets:
+    - 11 RNA-Seq and WXS PBTA samples (plus 2 WXS PBTA results from previous release)
+    - 1689 DGD targeted sequencing tumor samples
+    - methylation samples
+      - methylation samples: 
+        - 5 PBTA samples
+        - 12 TARGET samples
+      - methylation sample type not reported
+        - 3 PBTA 
+      - methylation tumor samples
+        - 1143 PBTA samples
+        - 751 TARGET samples
+      - RNA isoform merged TPM samples
+  - Update TCGA RNA-Seq data with GENCODE v36 GDC
+  - All additional changes as well as details about the above changes are as followed.
+
+- Add base histologies file to data release, see [ticket 333](https://github.com/PediatricOpenTargets/ticket-tracker/issues/333)
+- Update histologies file - key changes noted here (other tickets can be found in OpenPedCan repo)
+    - Change cancer_group of `Metastatic secondary tumors;Neuroblastoma` to `Neuroblastoma`
+            - See discussion in [ticket 232](https://github.com/PediatricOpenTargets/ticket-tracker/issues/232)
+            - [Ticket 234](https://github.com/PediatricOpenTargets/ticket-tracker/issues/234)
+    - Update `primary_site` for some GMKF and TARGET NBL samples to resolve lumped together `primary_site` issues
+            - [Ticket 257](https://github.com/PediatricOpenTargets/ticket-tracker/issues/257)
+    - Update histologies file to use `pathology_diagnosis` (when available) or `cancer_group` (when path dx not available) to fill in `harmonized_diagnosis` 
+            - [Ticket 259](https://github.com/PediatricOpenTargets/ticket-tracker/issues/259)
+     - DGD histologies cleanup: [ticket 325](https://github.com/PediatricOpenTargets/ticket-tracker/issues/325)
```

Comment:
```suggestion
```
We can remove this since it has never been released until now and was an upstream issue